packages/react-components/react-tree/src/hooks/useRovingTabIndexes.ts

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import { HTMLElementWalker } from './useHTMLElementWalker';
+
+interface RovingElement extends HTMLElement {
+  __fuiRoving: {
+    currentElement: HTMLElement;
+  };
```

Comment:
why do you need another object inside a ref? why can't the ref just store the current element?